lib/hanami/generators/action/view_spec.rspec.tt

```diff
@@ -7,6 +7,8 @@ RSpec.describe <%= config[:app] %>::Views::<%= config[:controller] %>::<%= confi
   let(:rendered)  { view.render }
 
   it 'exposes #foo' do
+    pending 'This is an auto-generated test. You should edit or delete it.'
+
     expect(view.foo).to eq exposures.fetch(:foo)
```

Comment:
@theCrab I think you should add the `# Example` here or delete above, we should have the same code.